packages/components/src/templates/next/layouts/Collection/filterUtils/getTagFilters.ts

```diff
@@ -0,0 +1,51 @@
+import type { Filter, FilterItem } from "../../../types/Filter"
+import type { ProcessedCollectionCardProps } from "~/interfaces"
+
+export const getTagFilters = (
+  items: ProcessedCollectionCardProps[],
+): Filter[] => {
+  // NOTE: Each tag is a mapping of a category to its
+  // associated set of values as well as the selected value.
+  // Hence, we store a map here of the category (eg: Body parts)
+  // to the number of occurences of each value (eg: { Brain: 3, Leg: 2})
+  const tagCategories: Record<string, Record<string, number>> = {}
+
+  items.forEach(({ tags }) => {
+    if (tags) {
+      tags.forEach(({ selected: selectedLabels, category }) => {
+        selectedLabels.forEach((label) => {
+          if (!tagCategories[category]) {
+            tagCategories[category] = {}
+          }
+          if (!tagCategories[category][label]) {
+            tagCategories[category][label] = 0
+          }
+
+          tagCategories[category][label] += 1
+        })
+      })
+    }
+  })
+
+  return Object.entries(tagCategories).reduce((acc: Filter[], curValue) => {
+    const [category, values] = curValue
+    const items: FilterItem[] = Object.entries(values).map(
+      ([label, count]) => ({
+        label,
+        count,
+        id: label,
+      }),
+    )
+
+    const filters: Filter[] = [
+      ...acc,
+      {
+        items,
+        id: category,
+        label: category,
+      },
+    ]
+
+    return filters
```

Comment:
issue: I think this is missing a sort function (both on the category and on the individual tags)

Reply:
this was copied over from `getCategories` from `utils` as-is so didn't touch it

but good idea to sort it, i will just add it in nevertheless

Reply:
addressed in https://github.com/opengovsg/isomer/pull/972/commits/6611f6e18e55931741f27e37b8f6c2b7df93e536